.env

```diff
@@ -43,3 +43,13 @@ LOCODE_DB_URL=https://github.com/nspcc-dev/neofs-locode-db/releases/download/v0.
 # NeoFS CLI binary
 NEOFS_CLI_URL=https://github.com/nspcc-dev/neofs-node/releases/download/v0.31.0/neofs-cli-amd64.tar.gz
 #NEOFS_CLI_PATH=/path/to/neofs-cli-binary
+
+# NeoFS ADM tool binary
+NEOFS_ADM_VERSION=v0.31.0
+NEOFS_ADM_URL=https://github.com/nspcc-dev/neofs-node/releases/download/${NEOFS_ADM_VERSION}/neofs-adm-amd64.tar.gz
+#NEOFS_ADM_PATH=/path/to/neofs-adm
+
+# Compiled NeoFS Smart Contracts
+NEOFS_CONTRACTS_VERSION=v0.15.5
+NEOFS_CONTRACTS_URL=https://github.com/nspcc-dev/neofs-contract/releases/download/${NEOFS_CONTRACTS_VERSION}/neofs-contract-${NEOFS_CONTRACTS_VERSION}.tar.gz
+#NEOFS_CONTRACTS_PATH=/path/to/neofs-contracts
```

Comment:
I think we should mention that local path is path to the repo, not to the tar archive.

Reply:
Agree, fixed.